pyvrp/diversity/_broken_pairs_distance.pyi

```diff
@@ -1,5 +1,36 @@
-from pyvrp import Individual, ProblemData
+from pyvrp._Individual import Individual
 
-def broken_pairs_distance(
-    data: ProblemData, first: Individual, second: Individual
-) -> float: ...
+def broken_pairs_distance(first: Individual, second: Individual) -> float:
```

Comment:
Can we add that this is directed broken pairs distance? (As we may want to have an undirected broken pairs distance for CVRP)?